lib/ukrand/Config.uk

```diff
@@ -0,0 +1,56 @@
+menuconfig LIKUKRAND
+	bool "ukrand: Software random number generator"
+	default n
+
+if LIKUKRAND
+
+config LIBUKSWRAND
+	bool "Software-generated randomness"
+	default n
+
+if LIBUKSWRAND
+
+choice
+	prompt "Algorithm"
+	default LIBUKSWRAND_CHACHA
+
+config LIBUKSWRAND_MWC
```

Comment:
Multiply-with-carry is not a CSPRNG so it should be deprecated: https://crypto.stackexchange.com/questions/10359/what-stops-the-multiply-with-carry-rng-from-being-a-cryptographically-secure-prn